Microsoft.Identity.Web/Client/TokenCacheProviders/CookieTokenCacheExtension.cs

```diff
@@ -0,0 +1,26 @@
+﻿using Microsoft.Extensions.DependencyInjection;
+
+namespace Microsoft.Identity.Web.Client
+{
+    using TokenCacheProviders;
+
+    /// <summary>
+    /// Extension class enabling adding the CookieBasedTokenCache implentation service
+    /// </summary>
+    public static class CookieTokenCacheExtension
```

Comment:
I've removed it in the Web App tutorial, @irvinesunday : the tokens tend to be too big for cookies in some cases (when the user belongs to a lot of group). This is not a reliable solution. Better remove this token cache implementation